curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/657f43d341a0dd17120cdb08.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-17
 
 # --description--
 
-Inside the space you left between the pair of square braces, you can describe the value that you would like to include in the list based on a given condition.
+Inside the pair of square braces, you can describe the value that you would like to include in the list based on a given condition.
 
 ```py
 snake_cased_char_list = [
```

Comment:
this needs to be expanded, this project really need a more comprehensive description of list comprehensions. It needs to go a bit deeper in the confusing mess that it is a list comprehension: it needs to mark the difference between just an expression, an if and an if/else, considering they need to be written in slighly different ways.

Reply:
Could you please provide some additional details on the areas where you feel further clarification is needed?

Reply:
So basically that's all the steps of the program?